packages/design-system/src/components/Modal/Modal.scss

```diff
@@ -0,0 +1,99 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+$modal-backdrop-z-index: 100;
+$modal-z-index: $modal-backdrop-z-index + 1;
+
+.modal-backdrop {
+	background-color: tokens.$coral-color-assistive-background;
+	opacity: tokens.$coral-opacity-l;
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	z-index: $modal-backdrop-z-index;
+}
+
+.modal {
+	position: absolute;
+	left: 50%;
+	top: 50%;
+	transform: translate(-50%, -50%);
+	z-index: $modal-z-index;
+	box-shadow: tokens.$coral-elevation-shadow-neutral-m;
+	border-radius: tokens.$coral-radius-s;
+	background: tokens.$coral-color-neutral-background;
+
+	> :first-child {
+		// Fixed dimensions have to be set on the flex element so that overflow-y: auto can
+		// properly compute the height of each component of the modal (header / content / buttons).
+		min-width: 660px;
+		max-width: 95vw;
+		max-height: 80vh;
+	}
+
+	&__header,
+	&__content,
+	&__buttons {
+		width: 100%;
+	}
+
+	&__content,
+	&__buttons {
+		padding: 0 tokens.$coral-spacing-xl;
+	}
+
+	&__header {
+		flex-shrink: 0;
+		height: 60px;
+		padding: tokens.$coral-spacing-xs tokens.$coral-spacing-l;
+		border-bottom: tokens.$coral-border-s-solid tokens.$coral-color-neutral-border-weak;
+
+		display: flex;
+		align-items: center;
+		column-gap: tokens.$coral-spacing-m;
+
+		&--with-tabs {
```

Comment:
There's currently no tab support in this modal AFAIK. If you need them now, then we need to update the Figma specs.

Reply:
Saw it in the examples, but indeed it didn't looked finished yet, removed the code related to that.